Userland/Libraries/LibWeb/Painting/PaintingCommandExecutorGPU.h

```diff
@@ -52,6 +52,8 @@ class PaintingCommandExecutorGPU : public PaintingCommandExecutor {
     virtual bool needs_prepare_glyphs_texture() const override { return true; }
     void prepare_glyph_texture(HashMap<Gfx::Font const*, HashTable<u32>> const&) override;
 
+    virtual void before_execute() override;
```

Comment:
Let's name this using some kind of verb, how about `prepare_to_execute()`?

Reply:
yes, that seems better. done!